<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinUIMoottoriDialog</class>
 <widget class="QDialog" name="PinUIMoottoriDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="PINsymbol_lineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="KeyPad">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>571</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(132, 132, 132);
border: 2px solid;
border-color: rgb(97, 97, 97);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pinButtonOk">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>401</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(63, 170, 63);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton8">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>291</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 6px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton6">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>181</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton7">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>291</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton3">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>71</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton9">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>291</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton5">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>181</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton1">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>71</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButtonClear">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>401</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(255, 30, 30);
font: 18pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton2">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>71</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton4">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>181</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton0">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>401</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(97, 97, 97);
border-radius: 8px;
background-color: rgb(221, 221, 221);
font: 26pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Syota_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>290</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>290</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>290</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Syötä PIN-koodi</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
